Sources/WebDriver/Session.swift

```diff
@@ -73,6 +73,14 @@ public class Session {
             Requests.SessionTimeouts(session: id, type: type, ms: duration * 1000))
     }
 
+    public func synchronousScript(javascript script: String,  args: [String]) throws {
```

Comment:
Does `synchronous(_:args:)` make more sense for this operation?

Reply:
I tried to follow the convention of being verbose with naming but I can make it more succinct if that would make more sense for the use

Reply:
Oh, that totally makes sense.  I think that we should fix the API surface to be more conventional for Swift rather than being verbose for the sake of being verbose.

Reply:
I think we were trying to mirror the webdriver API to a certain extent, to make things intuitive (guessable).

Reply:
Since they're the only two script functions on the webdriver API I think making them concise for reusability makes sense.